### PR TITLE
Change all occurrences of ExpressJS to Express

### LINKED ISSUE #962: Change All Occurrences of ExpressJS to Express
Express and ExpressJS has been used inconsistently in the code base. To make it consistent, change all occurrences of ExpressJS with Express.

### PR DESCRIPTION
**Pull request by streetcoder123**
Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/31781

**Changes**
- To be consistent, change from ExpressJS to Express

closes #962

### CODE REVIEW

**Review by piotrmark — DISMISSED** [review]

File: templates/_catalog/backendframeworks.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "summary": "JavaScript framework",
     "author": "Various",
     "order": "1",
-    "licenses": "[NodeJS](https://github.com/nodejs/node/blob/master/LICENSE)  \n[ExpressJS](https://github.com/expressjs/express/blob/master/LICENSE)  \n[ExpressJS Generator](https://github.com/expressjs/generator/blob/master/LICENSE)",
+    "licenses": "[NodeJS](https://github.com/nodejs/node/blob/master/LICENSE)  \n[Express](https://github.com/expressjs/express/blob/master/LICENSE)  \n[ExpressJS Generator](https://github.com/expressjs/generator/blob/master/LICENSE)",
```
Comment (jcoc611-microsoft) [inline templates/_catalog/backendframeworks.json]: Missed `ExpressJS Generator` here

File: src/client/src/mockData/mockVsCodeApi.ts [hunk 2]
```diff
@@ -145,20 +145,20 @@ const mockVsCodeApi = () => ({
                     order: 1,
                     metadataType: "Framework",
                     licenses:
-                      "[NodeJS](https://github.com/nodejs/node/blob/master/LICENSE)  \n[ExpressJS](https://github.com/expressjs/express/blob/master/LICENSE)  \n[ExpressJS Generator](https://github.com/expressjs/generator/blob/master/LICENSE)",
+                      "[NodeJS](https://github.com/nodejs/node/blob/master/LICENSE)  \n[Express](https://github.com/expressjs/express/blob/master/LICENSE)  \n[ExpressJS Generator](https://github.com/expressjs/generator/blob/master/LICENSE)",
```
Comment (jcoc611-microsoft) [inline src/client/src/mockData/mockVsCodeApi.ts]: Also `ExpressJS Generator` here

File: src/client/src/containers/QuickStart/defaultSelection.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ const BACK_END_SELECTION: ISelected = {
   author: "Various",
   internalName: "NodeJS",
   licenses:
-    "[NodeJS](https://github.com/nodejs/node/blob/master/LICENSE)  \n[ExpressJS](https://github.com/expressjs/express/blob/master/LICENSE)  \n[ExpressJS Generator](https://github.com/expressjs/generator/blob/master/LICENSE)",
+    "[NodeJS](https://github.com/nodejs/node/blob/master/LICENSE)  \n[Express](https://github.com/expressjs/express/blob/master/LICENSE)  \n[ExpressJS Generator](https://github.com/expressjs/generator/blob/master/LICENSE)",
```
Comment (jcoc611-microsoft) [inline src/client/src/containers/QuickStart/defaultSelection.ts]: Also `ExpressJS Generator` here

File: templates/_catalog/backendframeworks.json (hunk 1 above)
Comment (streetcoder123) [reply templates/_catalog/backendframeworks.json]: Oops missed that. Thanks :)

**Review by jasmingoh — APPROVED** [review]

**Review by jcoc611-microsoft — APPROVED** [review]

**Review by jsondoo — APPROVED** [review]
ship it

**Pull request merged**